pkg/workflows/workflow_test.go

```diff
@@ -744,3 +750,17 @@ func TestNewWorkflowLifecycle_Delete(t *testing.T) {
 	// Now try to delete
 	g.Expect(wfl.Delete(ctx, "addon-wf-test")).To(Not(HaveOccurred()))
 }
+
+func TestExtractAddOnNameAndLifecycleStep(t *testing.T) {
+	testsString := []string{
+		"event-router-4-prereqs-2c13ee7f-wf",
+		"event-router-4-install-2c13ee7f-wf",
+	}
+	for _, testsString := range testsString {
+		name, step, err := ExtractAddOnNameAndLifecycleStep(testsString)
+		if err != nil {
+			fmt.Printf("failed extracting addon name and lifecycle step.")
```

Comment:
This is a test, where's the assertion? I just see a print statement.

Reply:
db6f893de3812e442ece0edc1be00e9732a6902d